pkg/apply/apply.go

```diff
@@ -226,8 +226,14 @@ func (t *TopicApplier) applyExistingTopic(
 		return nil, err
 	}
 
-	changes, err := t.updateSettings(ctx, topicInfo)
-	if err != nil {
+	changes := &UpdateChangesTracker{
+		Action:      ActionEnumUpdate,
+		Topic:       t.topicName,
+		MissingKeys: make([]string, 0),
+		Error:       false,
```

Comment:
Out of curiosity -- is this ever not false?

Reply:
it can be if one of the update steps fails: https://github.com/getsentry/topicctl/blob/master/pkg/apply/apply.go#L235-L279